### PR TITLE
Enhancement #2081

### PR DESCRIPTION
**Pull request by lqdung-lockon**
I added a note: "「 * 」の 列 を 除去 す る こ と が で き ま す."
Changing conditions and check the upload: ['商品 削除 フ ラ グ', '送 料', '商品 規格 削除 フ ラ グ'] may not need to exist on a csv file.

### CODE REVIEW

File: src/Eccube/Resource/template/admin/Product/csv_product.twig [hunk 1]
```diff
@@ -99,6 +99,7 @@ $(function() {
             <div id="file_format_box__head" class="box-header">
                 <a href="{{ url('admin_product_csv_template', {'type': 'product'}) }}" id="download-button" class="btn btn-default pull-right btn-xs">雛形ファイルダウンロード</a>
                 <h3 class="box-title">商品登録CSVファイルフォーマット</h3>
+                <p class="help-block">「*」の列を除去することができます。</p>
```
Comment (hoand-vn) [inline src/Eccube/Resource/template/admin/Product/csv_product.twig]: @ryo-endo  please confirm about this changed GUI - we want explain more.

File: src/Eccube/Controller/Admin/Product/CsvImportController.php [hunk 2]
```diff
@@ -365,41 +360,31 @@ public function csvProduct(Application $app, Request $request)
                                     }
                                     $ProductClass = $this->createProductClass($row, $Product, $app, $data, $ClassCategory1, $ClassCategory2);
 
-                                    if ($BaseInfo->getOptionProductDeliveryFee() == Constant::ENABLED) {
```
Comment (hoand-vn) [inline src/Eccube/Controller/Admin/Product/CsvImportController.php]: @ryo-endo 
when delivery fee disable, we also insert this column into database:
1) admin can edit delivery fee to enable and use this column value
2) when disable this column will not affect to others.
Please confirm this solution. if NG, we will add it back.

**Comment by hoand-vn** [comment]
reivewed + tested with above assumptions : okie

**Comment by lqdung-lockon** [comment]
This issue merged into PR #42 . So it will be closed.

**Pull request closed without merging**